04/app.js

```diff
@@ -1,8 +1,38 @@
-console.log('DOM');
+console.log("DOM");
 
 // struktura do wykorzystania w pętli
 const menuItems = [
-    {text: 'start', url: '/'},
-    {text: 'galeria', url: '/gallery'},
-    {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+  { text: "start", url: "/" },
+  { text: "galeria", url: "/gallery" },
+  { text: "kontakt", url: "/contact" },
+];
+
+const nav = document.querySelector("nav");
+
+if (nav) {
+  const ul = document.createElement("ul");
+
+  //   ul.appendChild(createLinkItem("start", "/"));
+  //   ul.appendChild(createLinkItem("galeria", "/gallery"));
+  //   ul.appendChild(createLinkItem("kontakt", "/contact"));
+
+  menuItems.forEach((item) => {
+    if (item.text !== undefined && item.url !== undefined) {
```

Comment:
Bezpieczniej `typeof item.text !== 'undefined'` lub nawet `item.text` - `undefined` to wartość fałszywa więc zadziała o ile nie boimy się pustego ciągu znaków `''`